python/kfserving/server.py

```diff
@@ -0,0 +1,141 @@
+from http import HTTPStatus
```

Comment:
missing license header here?

Reply:
Do you know how to generate these with the hack file?